src/Features/TransientNotices.php

```diff
@@ -110,17 +110,15 @@ public static function remove( $notice_id ) {
 	}
 
 	/**
-	 * Enqueue notices to be displayed on page load.
+	 * Preload options to prime state of the application.
+	 *
+	 * @param array $options Array of options to preload.
+	 * @return array
 	 */
-	public static function enqueue_notices() {
-		$user_id = get_current_user_id();
-		$notices = self::get_queue_by_user( $user_id );
```

Comment:
Am I correct in thinking that this no longer enqueues notices for the current user? Would this introduce timing issues?

Reply:
Ah no, I can see in the next commit that the user check is happening on the client now. Carry on :-)